org.opent2t.sample.multisensor.superpopular/com.smartthings.sensorpod/js/tests/testdata.json

```diff
@@ -0,0 +1,19 @@
+{  "expected_exceptions" :
+    {
+        "GetAccelerationX" : {"statusCode" : 404, "messageConst" : "Error: Entity - undefined not found."},
```

Comment:
>messageConst [](start = 51, length = 12)

These need to be changed to "message" instead of "messageConst" since they aren't Error constants.